scripts/exrrfs_ioda_bufr.sh

```diff
@@ -17,23 +19,42 @@ yaml_list=(
 "prepbufr_aircar.yaml"
 #"prepbufr_aircft.yaml"
 #"prepbufr_ascatw.yaml"
-#"prepbufr_gpsipw.yaml"
 #"prepbufr_msonet.yaml"
 #"prepbufr_proflr.yaml"
 #"prepbufr_rassda.yaml"
-#"prepbufr_satwnd.yaml"
 #"prepbufr_sfcshp.yaml"
 #"prepbufr_vadwnd.yaml"
 )
 
-# run bufr2ioda.x
+# run bufr2ioda.x for prepbufr data
+# ---------------------------------
```

Comment:
Add an if switch after line 30 as follows:
```
if (( 1 == 2 )); then
```
So this block is turned off by default at the moment. We are not ready to test ztd and satwnd in realtime cycling runs at this time. Thanks!